### PR TITLE
Admin home page UI completed

### PR DESCRIPTION
**Pull request by atharva-borekar**
Implemented UI for admin home page where it shows all the drives separated into:
1. Ongoing Drives
2. Upcoming Drives
3. Completed Drives
Also implemented search to find a specific drive in the list of drives.
Need to redirect to different component onclick of 'Candidates' and 'Edit'

### CODE REVIEW

File: src/modules/admin/adminHome/AdminHomeContainer/getDriveDetails.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { Alert } from 'core-components';
+
+import { get } from 'redux/admin/apiHelper';
+import { SERVER_URL } from 'constants/appConstants';
+
+const getDriveDetails = async () => {
+  const drives = await get(`${SERVER_URL}api/v1/admin/drives`)
```
Comment (Suhas010) [inline src/modules/admin/adminHome/AdminHomeContainer/getDriveDetails.js:7]: Incorrect async-await usage. please refer [this](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await#adding_error_handling) example for modification.

File: src/modules/admin/adminHome/AdminHomeContainer/index.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+              <Button>Candidates</Button>
+            </td>
+          </tr>
+        );
+      }
+      return false;
+    });
+  }, [driveDetailsData.upcomingDrives]);
+
+  const renderCompletedDrives = useMemo(() => {
```
Comment (Suhas010) [inline src/modules/admin/adminHome/AdminHomeContainer/index.jsx:73]: `renderOngoingDrives `, `renderUpcomingDrives` and  `renderCompletedDrives` duplications

File: src/modules/admin/adminHome/AdminHomeComponent/index.jsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            }}
+          >
+            Create Drive
+          </Button>
+        </Col>
+      </Row>
+      <Row className='py-2'>
+        <Col xs={10} lg={10} xl={10}>
+          <thead>
+            <h3 className='px-3'>Ongoing Drives</h3>
```
Comment (Suhas010) [inline src/modules/admin/adminHome/AdminHomeComponent/index.jsx:50]: Can we write a common function to render for both the rows,
`Ongoing drives` and `upcoming drives` Only title is differenct

**Review by Suhas010 — CHANGES_REQUESTED** [review]
Please implement commented changes.

File: src/modules/admin/adminHome/AdminHomeContainer/getDriveDetails.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
+import { Alert } from 'core-components';
+
+import { get } from 'redux/admin/apiHelper';
+import { SERVER_URL } from 'constants/appConstants';
+
+// const getDriveDetails = async () => {
```
Comment (shailendrakanherkar18) [inline src/modules/admin/adminHome/AdminHomeContainer/getDriveDetails.js:6]: Why we still have commented code?

File: src/modules/admin/adminHome/AdminHomeContainer/index.jsx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+import getDriveDetails from 'modules/admin/adminHome/AdminHomeContainer/getDriveDetails';
+import { useDispatch } from 'react-redux';
+
+const CreateDriveContainer = () => {
+  const [driveDetailsIsLoading, setDriveDetailsIsLoading] = useState(true);
+  const [driveDetailsData, setDriveDetailsData] = useState([]);
+
+  const [queryIsLoading, setQueryIsLoading] = useState(true);
+
+  const [q, setQ] = useState('');
```
Comment (shailendrakanherkar18) [inline src/modules/admin/adminHome/AdminHomeContainer/index.jsx:15]: what is q ?? 
Can we meaningful name ?

File: src/modules/admin/home/HomeContainer/adminHomeComponentReducer.js [hunk 6]
```diff
@@ -1,18 +1,28 @@
 export const initialHomeState = {
-  currentScreen:'HOME',
+  currentScreen: 'HOME',
+  id: '',
 };
 
-export const adminHomeComponentReducer = (state = initialHomeState, action = {}) => {
+export const adminHomeComponentReducer = (
```
Comment (shailendrakanherkar18) [inline src/modules/admin/home/HomeContainer/adminHomeComponentReducer.js:6]: why we dont have immutability added here ?

**Review by shailendrakanherkar18 — APPROVED** [review]

**Pull request merged**